### ISSUE TITLE
Uncaught TypeError when using numeric()

### ISSUE THREAD

**Issue by apandichi** [KevinSheedy/jquery.alphanum#32]
If I modify the file https://github.com/KevinSheedy/jquery.alphanum/blob/master/index.html as follows
```js
		$('#accountBalance').numeric({
			maxPreDecimalPlaces : 10,
			maxDecimalPlaces    : 2
		});
```
to
```js
		$('#accountBalance').numeric();
```
The input text will throw an error on blur event:
```
Uncaught TypeError: Cannot read property 'min' of undefined 
jquery.alphanum.js:218
numericField_Blur jquery.alphanum.js:218
(anonymous function) jquery.alphanum.js:41
f.event.dispatch jquery.js:3
f.event.add.h.handle.i jquery.js:3
```

**Comment by KevinSheedy** [comment]
Fixed in 1.0.23

**Comment by KevinSheedy** [comment]
Thanks @leesaxby for the PR
